nengo_gui/static/menu.js

```diff
@@ -27,6 +27,8 @@ Nengo.Menu.prototype.show = function (x, y, items) {
         return;
     }
 
+    this.menu_margin = 15;
+
```

Comment:
(Sorry, to be more clear in my comment, why is there a ```menu_margin``` at all?  Do other browsers do that?  I don't see that effect in either Chrome or Firefox on my computer....)